src/device/pf_pdport.c

```diff
@@ -233,6 +263,20 @@ static int pf_pdport_load (pnet_t * net, int loc_port_num)
          pf_lldp_send_enable (net, loc_port_num);
       }
 
+      if (p_port_data->pdport.adjust.mask & PF_PDPORT_ADJUST_DCPB_MASK)
+      {
+	 int bident = p_port_data->pdport.adjust.dcp_boundary.dcp_boundary.do_not_send_dcp_ident;
+	 int bhello = p_port_data->pdport.adjust.dcp_boundary.dcp_boundary.do_not_send_dcp_hello;
+
+         pf_pdport_dcp_boundary (net, loc_port_num, 0x00, bident);
+         pf_pdport_dcp_boundary (net, loc_port_num, 0x01, bhello);
+      }
+      else
+      {
+         pf_pdport_dcp_boundary (net, loc_port_num, 0x00, 0);
```

Comment:
not a big thing, but maaaybe would be more readable to use some defines for 0x00 and 0x01 (i.e. DCP_BOUND_TYPE).

Reply:
Similar for the last value in the function (BLOCK | ALLOW)

Reply:
Only for readability... Otherwise, you can ignore these comments

Reply:
Great stuff, I'll change the 0x00 and 0x01 to `DCP_IDENT_GROUP` and `DCP_HELLO_GROUP`, and use a boolean true/false for block/unblock perhaps?

Reply:
Perfect